packages/transaction-backend/package.json

```diff
@@ -16,11 +16,14 @@
     "copy-files": "cp ./src/openapi.yml ./build/",
     "test": "jest --runInBand",
     "verify": "tsc --noEmit",
-    "serve": "nodemon"
+    "serve": "nodemon",
+    "get-buy-tx": "npx ts-node scripts/gettxbuy.ts",
+    "get-orders": "npx ts-node scripts/getorders.ts"
```

Comment:
don't use npx inside scripts
add ts-node to devDeps and use ts-node without npx